src/components/EventDetails/EventDetails.js

```diff
@@ -105,32 +112,51 @@ class EventDetails extends Component {
           </div>
 
           <div className={styles.right}>
-            <div className={styles.eventLink}>
-              <a
-                href={browserUrl}
-                target="_blank"
-                rel="noopener noreferrer"
-                className={styles.shareBtn}
-              >
-                EVENT PAGE
-              </a>
-            </div>
-            {loc && renderAddress(loc)}
-            {startDate && renderTimeRange(startDate, endDate)}
-            <div className={styles.socialBtns}>
-              <button onClick={this.showHideSocialBtns}>
-                SHARE
-              </button>
-              <div className={this.state.socialBtnsDisplay}>
-                <a href={shareUrl} className={styles.facebookBtn}>
-                  <FaFacebookSquare size={25} />
-                  <span>Share on Facebook</span>
-                </a>
-                <a href={shareUrl} className={styles.twitterBtn}>
-                  <FaTwitter size={25} />
-                  <span>Share on Twitter</span>
+            <div className={styles.infoLinks}>
+              <div className={styles.eventLink}>
+                <a
+                  href={browserUrl}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  className={styles.eventBtn}
+                >
+                  <span>
+                    FACEBOOK&nbsp;
+                  </span>
+                  EVENT PAGE
                 </a>
               </div>
+              <div className={styles.sharing}>
+                <div
+                  className={styles.shareBtnMobile}
+                  onClick={this.toggleSocialPopup}
```

Comment:
Good call, Pat! I just added `role="link"` to this div so that screen readers would see it as a link.